glambie/plot/processing_plots.py

```diff
@@ -0,0 +1,383 @@
+    plot_raw_and_homogenized_input_data_of_data_group(catalogue_raw=data_catalogue_trends_raw,
+                                                      catalogue_homogenized=data_catalogue_trends_homogenized,
+                                                      trend_combined=timeseries_trend_combined,
+                                                      data_group=data_group,
+                                                      region=region,
+                                                      category="trends",
+                                                      output_filepath=plot_fp)
+    plot_fp = output_path_handler.get_plot_output_file_path(region=region, data_group=data_group,
+                                                            plot_file_name="7_trends_recalibration.png")
+    plot_recalibration_of_annual_variability_with_trends(catalogue_trends=data_catalogue_trends_homogenized,
+                                                         catalogue_calibrated_series=data_catalogue_calibrated_series,
+                                                         trend_combined=timeseries_trend_combined,
+                                                         data_group=data_group,
+                                                         region=region,
+                                                         output_filepath=plot_fp)
+    plot_fp = output_path_handler.get_plot_output_file_path(region=region, data_group=data_group,
+                                                            plot_file_name="8_combined_result.png")
+    plot_recalibrated_result_of_data_group(catalogue_trends=data_catalogue_trends_homogenized,
+                                           catalogue_calibrated_series=data_catalogue_calibrated_series,
+                                           trend_combined=timeseries_trend_combined,
+                                           data_group=data_group,
+                                           region=region,
+                                           output_filepath=plot_fp)
+
+
+def plot_raw_input_data_of_data_group(catalogue_raw: DataCatalogue,
+                                      trend_combined: DataCatalogue,
+                                      data_group: GlambieDataGroup,
+                                      region: RGIRegion,
+                                      category: str,
+                                      output_filepath: str):
+    _, ax = plt.subplots(2, 1, figsize=(7, 6))
+
+    count = 0
+    for _, ds in enumerate(catalogue_raw.datasets):
+        for _, row in ds.data.as_dataframe().iterrows():
+            time_period = row["end_dates"] - row["start_dates"]
+            ax[0].plot([row["start_dates"], row["end_dates"]], [np.array(row["changes"]) / time_period,
+                                                                np.array(row["changes"]) / time_period],
+                       color=COLOURS[count])
```

Comment:
Thanks, that was just laziness on my part ;) I adapted it now so that it uses the first few colours as defined from list and any additional ones from a cmap. Not the most elegant solution but does the job.

Reply:
Lovely! Thank youu